rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -143,24 +164,23 @@ void SetupResumeAcceptor::processFrame(
         break;
       }
 
-      if (!onResume(transport, std::move(resumeParams))) {
+      try {
+        onResume(transport, std::move(resumeParams));
+      } catch (RSocketException ex) {
         transport->outputFrameOrEnqueue(frameSerializer->serializeOut(
-            Frame_ERROR::rejectedResume("Resumption Failure")));
+            Frame_ERROR::rejectedResume(ex.what())));
```

Comment:
Not sure about this, this has the potential to leak information. Could we make a specific type that has a "user-friendly" message and use that?